README.md

```diff
@@ -203,7 +203,7 @@ app.use(expressValidator({
       return new Promise(function(resolve, reject) {
         User.findOne({ username: username })
         .then(function(user) {
-          if (user) {
+          if (!user) {
             resolve(user);
```

Comment:
What do you think of even removing the `user` parameter from this `resolve()` call?
It doesn't make much sense to resolve something which is probably undefined :)